tmt/steps/provision/artemis.py

```diff
@@ -736,6 +751,12 @@ class ProvisionArtemis(tmt.steps.provision.ProvisionPlugin[ProvisionArtemisData]
             image: Fedora
             api-url: https://your-artemis.com/
 
+    .. note::
+
+        When used together with TF infrastructure
```

Comment:
```suggestion
        When used together with the Testing Farm infrastructure
```

Let's prevent using too much abbreviations in the docs.

Reply:
Similar below.

Reply:
Updated !